### ISSUE TITLE
[feature]: add basic smoke tests for linux and macOS

### ISSUE THREAD

**Issue by slayerjain** [keploy/keploy#2242]
### 👀 Is there an existing feature request for this?

- [X] I have searched the existing issues

### 🔖 Enhancement description

It would be nice to have some basic smoke tests that checks if the installation command is working fine

### 🎤 Why should this be worked on?

Validating if the 1-click is working should happen after the release process is complete. We should add trigger once the release and docker build is complete. 

We need validate for linux native, linux docker, macOS native,  macOS docker, windows native, windows docker. 

### 💻 Repository

keploy

**Comment by slayerjain** [comment]
we can start with macOS  and linux before windows

**Comment by 047pegasus** [comment]
I can take this up. I understand these platforms and will be happy to help. Assign it to me please.

**Comment by Tanuj1718** [comment]
@Swpn0neel @slayerjain   If it is still open, can you please assign it to me?

**Comment by samikshadeore42** [comment]
Can you assign mme for this issue

**Comment by AnkitPadhiyar** [comment]
Hello,
Our team (T186) would like to take up this issue. Kindly assign it to us.
Thanks!
